src/index.html

```diff
@@ -1,13 +1,48 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;600&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body data-qa="card">
+    <img class="imac-img" src="./images/imac.jpeg"
+          alt="imac">
+    <p class="title-text">
+      APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+    </p>
+    <p class="code">
+      Product code: 195434
+    </p>
+
+    <div class="container">
+      <div class="container-stars">
+        <img src="./images/star-active.svg" alt="star-active">
+        <img src="./images/star-active.svg" alt="star-active">
+        <img src="./images/star-active.svg" alt="star-active">
+        <img src="./images/star-active.svg" alt="star-active">
+        <img src="./images/star.svg" alt="star">
+      </div>
+      <p class="reviews">
+        Reviews: 5
+      </p>
+    </div>
+    <div class="container-price">
+      <p class="price">
+        Price:
+      </p>
+      <p class="bill">
+        $2,199
+      </p>
+    </div>
+    <div class="container-button">
+      <button type="button" class="buy-button"
+             data-qa="hover">BUY</button>
+    </div>
```

Comment:
You are not following the BEM methodology in your code at all. Please read about BEM and follow its rules. Also pay attention to the 'alt' attribute usage, this attribute should describe the image content (you can find good examples at style guide